CHANGELOG.md

```diff
@@ -5,7 +5,12 @@ to [Semantic Versioning][].
 
 This document follows the conventions laid out in [Keep a CHANGELOG][].
 
-## [Unreleased][]
+## [Unreleased][2.0.5] [2.0.5][2.0.4-2.0.5] - 2023-03-15

```

Comment:
Remove 2.0.5 pls from this file.

Only changes are needed to documentation an seperate release commit will be used to create an release.